src/Symfony/Component/Debug/ExceptionHandler.php

```diff
@@ -411,4 +414,27 @@ private function getSymfonyGhostAsSvg()
     }
+
+    private static function getTraceClassname($trace)
+    {
+        if (empty($trace['file'])) {
+            return null;
+        }
+
+        $vendorRoot = static::getVendorRoot();
+        $traceFileRelative = str_replace($vendorRoot, '', realpath($trace['file']));
+
+        if (strpos($traceFileRelative, '/vendor/') === 0) {
```

Comment:
@rentalhost sorry for the delay. The code to look for vendors in LinkStub handles two things yours doesn't:
- there can be several vendor directories - eg the global one, the app's, phpunit's when running test, to consider a not uncommon situation
- the vendor directories can have a [different name than "vendor"](https://getcomposer.org/doc/06-config.md#vendor-dir)